README.rst

```diff
@@ -0,0 +1,119 @@
+tool which is capable of understanding all of them would be welcomed by
+science developers.
+
+Thus a tool was developed based on a framework which hopes to support ready
+extension and reconfiguration.
+
+The project is still in its infancy so only a few rules have been implemented
+and only for Fortran. Stubs are provided to show how it might be extended to
+support C.
+
+Find the project at <https://github.com/MetOffice/stylist>
+
+.. |BSD3 License| image:: https://img.shields.io/badge/License-BSD_3--Clause-blue.svg
+   :target: https://opensource.org/licenses/BSD-3-Clause
+
+.. |GitHub release| image:: https://img.shields.io/github/release/MetOffice/stylist.svg
+   :target: https://github.com/MetOffice/stylist/
+
+.. |PyPI version| image:: https://badge.fury.io/py/stylist.svg
+   :target: https://pypi.python.org/pypi/stylist/
+
+.. |GitHub merge testing| image:: https://github.com/MetOffice/stylist/workflows/Merge%20Test/badge.svg
+   :target: https://github.com/MetOffice/stylist/actions
+
+
+Installation
+~~~~~~~~~~~~
+
+Installation can be as simple as ``pip install stylist`` or
+``conda install -c conda-forge stylist``.
+
+As always it is also possible to install from the project source by running
+``python setup.py``. The source may be obtained by downloading a tarball or by
+cloning the repository.
+
+
+Usage
+~~~~~
+
+Stylist provides a command-line tool ``stylist`` for normal use. It can also be
+used as a package if you want to integrate it with another tool. Documentation
+regarding this second option is maintained in the project wiki.
+
+On the Command Line
+-------------------
+
+The command-line tool is not complicated to use::
+
+  stylist [-help] [-verbose]
+          [-configuration FILENAME]
+          [-style NAME]
+          FILE ...
+
+The only required arguments are a configuration file and one or more
+filenames. These are the files which will be checked. If a directory is
+specified then the tool will automatically descend into it checking all files
+which it recognises by extension.
+
+If you want a running commentary of what the tool is doing then use the
+``-verbose`` argument.
+
+A configuration file may specified with ``-configuration`` . This file should
+be formatted as documented below. There must be a configuration file which
+defines at least one style. While the fallback mechanism which allows for user
+and site configurations has not been implemented it must be specified with this
+argument.
+
+The configuration may define several styles, in which case one can be chosen
+using the ``-style`` argument. If it is not then the first in the configuration
+file will be used.
+
+Configuration File
+------------------
+
+The configuration file is a python script.
+
+The processing pipelines for different file types are specified with
+``FilePipe`` objects. The variable to which it is assigned names the
+file name extension it refers to. The first argument is the language source
```

Comment:
"The variable to which it is assigned names the
file name extension it refers to" Maybe this specific part could use an example, it's a bit wordy and hard to understand, and i feel like if you gave some context it would make more sense